Cyberpunk2k20_v2/Cyberpunk2k20.html

```diff
@@ -0,0 +1,3893 @@
+            -->
+            <h4>Class:</h4>
+            <span class="bio-title" class="bioView" name="attr_class"></span>
+            <input class="bioEdit" type="text" name="attr_class" style="width:150px;">
+            <h4>Genre:</h4>
+            <span class="bio-title" class="bioView" name="attr_sex"></span>
+            <select name="attr_sex" style="width:70px;">
+                <option value="M">Male</option>
+                <option value="F">Female</option>
+                <option value="O">Other</option>
+            </select>
+            <!--
+            <select class="bioEdit" name="attr_class" style="width:180px;">
+                <option value="Solo">Solo</option>
+                <option value="Cop">Cop</option>
+                <option value="Corp">Corp</option>
+                <option value="Fixer">Fixer</option>
+                <option value="Techie">Techie</option>
+                <option value="Rockerboy/girl">Rockerboy/girl</option>
+                <option value="Nomad">Nomad</option>
+                <option Value="Netrunner">Netrunner</option>
+                <option Value="Medtech">Medtech</option>
+                <option Value="Media">Media</option>
+                <option value="Covert Mission Spe">Covert Mission Spe</option>
+                <option value="Panzerboy">Panzerboy</option>
+                <option value="Dancer/Prostitute">Dancer/Prostitute</option>
+                <option value="Con Man">Con Man</option>
+                <option value="Prowler">Prowler</option>
+                <option value="Aerojock">Aerojock</option>
+                <option value="Assassin">Assassin</option>
+                <option value="Nomad Warrior">Nomad Warrior</option>
+                <option value="Runner">Runner</option>
+                <option value="Politician">Politician</option>
+            </select>
+            -->
+            <h4>Age:</h4>
+            <span class="bio-title" name="attr_age"></span>
+            <input type="number" name="attr_age" value="0" />
+            <h4>IP:</h4>
+            <span class="bio-title" name="attr_ip"></span>
+            <input type="number" name="attr_ip" value="0" style="width:50px;" />
+            <h4>&nbsp;&nbsp;Cash:</h4>
+            <input type="number" name="attr_money" value="0" style="width:100px;border:0;text-align:right;display:inline-block;" /><i style="margin-top:5px;font-size:smaller;">€$</i>
+        </div>
+        <div class="SNS dmg-calculator">
+            ##TODO##
+        </div>
+        <div class="FFN dmg-calculator">
+            <h4>Combat (damages calculator & random location) </h4>
+            <button style="margin-left:50px;" class="add-dice" type="roll" value="&amp;{template:skill}{{name=@{character_name}}}{{skillname=Initiative}}{{result=[[1d10!cf<1+@{ref}+@{ref_mod}-@{ref_penality}+@{init_mod}+?{Type|Normal,0|Quick,3}+?{mod.|0}]]}}{{mod=?{mod.|0}}}" name="roll_init"><strong>Initiative</strong></button>
```

Comment:
Per Building Character Sheet standards repository sheets must have [standalone functionality and cannot require APIs by default.](https://wiki.roll20.net/Building_Character_Sheets#3._A_Satisfactory_Experience). APIs functionality can be added as an optional feature which can be toggled one. The Forbidden Land sheet is a good example of how this works.

Reply:
Normally I have put 2 buttons, one for normal sheet and the second for API work. But maybe there is a better way to do the same.